bin/provisioner.js

```diff
@@ -1 +1,69 @@
-//TODO: The same as server but without configuring api...
```

Comment:
If you look at the scripts I have else where they actually use `module.parent` to detected if they are top-level and only run `lunch` in that case... otherwise they just export the `launch` function.

This makes it possible to run it within the same process in the tests.